cmd/cluster/log-exporter/query_log_exporter.go

```diff
@@ -68,13 +68,36 @@ var enableDbQueryLoggingCmd = &cobra.Command{
 
 		resp, r, err := authApi.EnableDbQueryLogging(clusterId).PgLogExporterConfigSpec(
 			ybmclient.PgLogExporterConfigSpec{ExportConfig: exportConfig, ExporterId: integrationId}).Execute()
+		dqlConfig := resp.GetData()
 
 		if err != nil {
 			logrus.Debugf("Full HTTP response: %v\n", r)
 			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
 		}
 
-		formatter.DbQueryLoggingWriteFull(resp.GetData(), integrationName)
+		msg := fmt.Sprintf("The db query logging for cluster %s is being enabled", clusterName)
+		if viper.GetBool("wait") {
+			returnStatus, err := authApi.WaitForTaskCompletion(clusterId, ybmclient.ENTITYTYPEENUM_CLUSTER, ybmclient.TASKTYPEENUM_ENABLE_DATABASE_QUERY_LOGGING, []string{"FAILED", "SUCCEEDED"}, msg)
```

Comment:
This part of the code is repetitive, can you create one function, something like `waitForDbLoggingTaskCompletion` 
and common out this code